extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/FileTlsTrustManagersProvider.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.amazon.common.runtime;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import software.amazon.awssdk.http.TlsTrustManagersProvider;
+
+public class FileTlsTrustManagersProvider implements TlsTrustManagersProvider {
```

Comment:
Please add javadoc here explaining the purpose of this class

Reply:
Will do it.